src/wp-content/themes/twentyfifteen/inc/customizer.php

```diff
@@ -792,10 +793,23 @@ function twentyfifteen_color_scheme_css_template() {
 		'secondary_sidebar_textcolor' => '{{ data.secondary_sidebar_textcolor }}',
 		'meta_box_background_color'   => '{{ data.meta_box_background_color }}',
 	);
-	?>
-	<script type="text/html" id="tmpl-twentyfifteen-color-scheme">
-		<?php echo twentyfifteen_get_color_scheme_css( $colors ); ?>
-	</script>
-	<?php
+
+	$css = twentyfifteen_get_color_scheme_css( $colors );
+
+	if ( function_exists( 'wp_print_inline_script_tag' ) ) {
+		wp_print_inline_script_tag(
+			$css . "\n//# sourceURL=" . __FUNCTION__,
```

Comment:
This is actually not CSS but rather an HTML template for constructing CSS, so it would not get a `sourceURL` comment, I believe.

```suggestion
			$css,
```